dvc/dvcfile.py

```diff
@@ -65,7 +65,7 @@ def is_lock_file(path):
 def check_dvc_filename(path):
     if not is_valid_filename(path):
         raise StageFileBadNameError(
-            "bad DVC-file name '{}'. DVC-files should be named "
+            "bad DVC file name '{}'. DVC files should be named "
             "'Dvcfile' or have a '.dvc' suffix (e.g. '{}.dvc').".format(
```

Comment:
I think we can get rid of the `Dvcfile` part of this error message entirely since we are completely dropping single-stage support in 2.0?

Reply:
I would leave it as it is for now. Later it can be changed closely to 2.0. Anyway, I'm not so informed about 2.0 specific, so it could be difficultly to me to change the message.

Reply:
The current master branch is for 2.0 so it is safe to change it. The message should be something like:

`bad DVC file name '{}'. DVC files should be named '{PIPELINE_FILE}' or have a '.dvc' suffix (e.g. '{}.dvc')`